qan/slowlog/slowlog_test.go

```diff
@@ -365,17 +363,6 @@ func (s *WorkerTestSuite) TestRotateAndRemoveSlowLog(t *C) {
 		t.Error("/tmp/" + slowlogFile + " no longer exists")
 	}
 
-	// The original slow log should have been renamed to slow006-TS, parsed, and removed.
-	files, _ = filepath.Glob("/tmp/" + slowlogFile + "-[0-9]*")
-	if len(files) != 0 {
-		t.Errorf("Old slow log removed, got %+v", files)
-	}
-	defer func() {
-		for _, file := range files {
-			os.Remove(file)
-		}
-	}()
```

Comment:
This is probably related to the fact that we don't support `RemoveOldSlowLogs` variable anymore